Code/src/megadesk.cpp

```diff
@@ -724,18 +739,13 @@ void loop()
   {
     memoryMoving = false;
     // max() allows escape from recalibration (below DANGER_MIN_HEIGHT)
-    targetHeight = max(currentHeight + HYSTERESIS + 1, DANGER_MIN_HEIGHT);
+    targetHeight = max(currentHeight + MOVE_OFFSET, DANGER_MIN_HEIGHT);
   }
   else if (manualMove == Command::DOWN)
   {
     memoryMoving = false;
     // min() allows descend if beyond DANGER_MAX_HEIGHT
-    targetHeight = min(currentHeight - HYSTERESIS - 1, DANGER_MAX_HEIGHT);
-  }
-  else if (!memoryMoving)
-  {
```

Comment:
Completely removed resetting of targetHeight here - may still be de-accelerating.